spec/ingram_micro/elements/sales_order_header_spec.rb

```diff
@@ -6,6 +6,7 @@
 
     context 'when using the domestic schema' do
       it "returns the correct default values hash" do
+        IngramMicro.configuration.international = false
```

Comment:
not sure where the code that adds `international` key, it should be renamed to `international_schema`